components/whyd/2022/Terminal/UserInputArea.vue

```diff
@@ -139,13 +147,12 @@ export default {
     optionButtonClick(_, commandKey) {
       // event parameter is discarded
       // const command = this.terminalCommands[commandKey]
-      this.$emit('addTextLine', { content: `> ${commandKey}`, class: '' })
+      this.$emit('addTextLine', { content: `> ${commandKey}` })
       this.processCommand(commandKey)
       this.scrollToBottom()
     },
     processCommand(input) {
-      const processed = input.trim().toLowerCase()
-      console.log(processed)
+      const processed = input.trim() // .toLowerCase() fuck you if you try to use caps in the terminal
```

Comment:
😏 

good point though, it may be a good idea to make the terminal case-insensitive since it's just a toy

Reply:
shit i didn't mean to make this a review

Reply:
I had to change this because I realized that the user could not cd to whyd/SecBot without it haha